provisioning/Samples/device/ComputeDerivedSymmetricKeySample/ComputeDerivedSymmetricKeySample.csproj

```diff
@@ -2,8 +2,8 @@
 
   <PropertyGroup>
 	  <OutputType>Exe</OutputType>
-	  <TargetFramework>netcoreapp2.1</TargetFramework>
-	  <LangVersion>8.0</LangVersion>
+	  <TargetFramework>net5.0</TargetFramework>
+    <LangVersion>8.0</LangVersion>
```

Comment:
This lost its indentation

Reply:
Actually there are tabs in this file making it odd. If you change it all to spaces, it'll be fine.